dist/package/openSUSE-release-tools.spec

```diff
@@ -58,6 +58,10 @@ Requires:       python3-pycurl
 Requires:       python3-python-dateutil
 Requires:       python3-pyxdg
 Requires:       python3-requests
+# typing extensions are needed on SLE & Leap
+%if 0%{?suse_version} < 1600
```

Comment:
Is it in ALP? If yes, can be changed to `<= 1500`

Reply:
I hope that alp is at python >= 3.8! Then it wouldn't be required.

Reply:
Should be 3.11

Reply:
Fixed